examples/04_vector_field/01_3d_vector_field.py

```diff
@@ -0,0 +1,70 @@
+"""
+Generating a Random 3D Vector Field
+-----------------------------------
+
+In this example we are going to generate a random 3D vector field with a
+Gaussian covariance model. The mesh on which we generate the field will be
+externally defined and it will be generated by PyVista.
+"""
+import numpy as np
+import gstools as gs
+import pyvista as pv
+
+pv.set_plot_theme("document")
+
+###############################################################################
+# create a uniform grid with PyVista
+nx, ny, nz = 40, 30, 10
+mesh = pv.UniformGrid((nx, ny, nz), (1., 1., 1.), (-10., 0., 0.))
+x = mesh.points[:, 0]
+y = mesh.points[:, 1]
+z = mesh.points[:, 2]
+
+###############################################################################
+# create an incompressible random 3d velocity field on the given mesh
+model = gs.Gaussian(dim=3, var=3, len_scale=1.5)
+srf = gs.SRF(model, generator='VectorField')
+srf((x, y, z), mesh_type='unstructured', seed=198412031)
+
+# add a mean velocity in x-direction
+srf.field[0,:] += 0.5
+
+###############################################################################
+# add the velocity field to the mesh object
+mesh["Velocity"] = srf.field.T
```

Comment:
Why not using the `mesh` method? And set seed in `__init__` so sampling is done only once,
```python
srf = gs.SRF(model, generator='VectorField', seed=198412031)
srf.mesh(mesh, points="points", name="Velocity")
mesh["Velocity"][:, 0] += 0.5
```

Reply:
:-D I'm simply not keeping up with the development of GSTools... But I think the method is not working with vector fields... I'm gonna test that again and open an issue if that is true.
I suggest to keep the example as it is for the moment and update it, as soon as I figured out the problem.

Reply:
You are right... it's not working with Vector fields. Damn.. please create an issue.

Reply:
Will do, but I won't make it today anymore...